azurerm/resource_arm_key_vault_certificate.go

```diff
@@ -236,6 +239,11 @@ func resourceArmKeyVaultCertificate() *schema.Resource {
 				Computed: true,
 			},
 
+			"thumbprint": {
+				Type:     schema.TypeString,
+				Computed: true,
+			},
```

Comment:
could we [add this field to the documentation](https://github.com/terraform-providers/terraform-provider-azurerm/blame/master/website/docs/r/key_vault_certificate.html.markdown#L251)?

Reply:
on it